MSAL/test/integration/native_auth/end_to_end/sign_up/MSALNativeAuthSignUpUsernameEndToEndTests.swift

```diff
@@ -184,8 +191,13 @@ final class MSALNativeAuthSignUpUsernameEndToEndTests: MSALNativeAuthEndToEndBas
 
         let submitCodeExp = expectation(description: "submit code")
         let signUpVerifyCodeDelegate = SignUpVerifyCodeDelegateSpy(expectation: submitCodeExp)
+        
```

Comment:
Same as my other comment, but for the line 181 (sorry, Github doesn't allow me to write a comment in that line)